### ISSUE TITLE
[macOS] Command CodeSign failed with a nonzero exit code

### ISSUE THREAD

**Issue by jinfagang** [flutter/flutter#74815]
I am try build macos release, I have using xcode set sign, still got this error:

```
daybreak_flutter/build/macos/Build/Products/Release/daybreak_flutter.app: bundle format unrecognized, invalid, or unsuitable
In subcomponent: /U_flutter/build/macos/Build/Products/Release/daak_flutter.app/Contents/Frameworks/App.framework
Command CodeSign failed with a nonzero exit code
note: Using new build system
note: Building targets in parallel
note: Planning build
note: Constructing build description
** BUILD FAILED **
```

what could be the reason?

**Comment by darshankawar** [comment]
@jinfagang 
Can you provide `flutter doctor -v` ?
There was similar issue [fixed](https://github.com/google/flutter-desktop-embedding/issues/487) in `flutter-desktop-embedding` as well as in flutter repo, https://github.com/flutter/flutter/issues/38728

**Comment by jinfagang** [comment]
@darrenaustin I am using the latest flutter in master branch.

I thought it maybe my sign not right but after I fix the sign it still got this issue, seems not effected.

**Comment by darshankawar** [comment]
@jinfagang 
Can you elaborate on how did you setup the sign in Xcode, what fix did you make to it alongwith `flutter build macos --verbose` log ?

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
